<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/nematizadeh/.designer/backup/app.png</normaloff>
    <normalon>app.png</normalon>C:/Users/nematizadeh/.designer/backup/app.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(6, 14, 20);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 30, 44);</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>-30</y>
       <width>311</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 900 14pt &quot;Segoe UI Black&quot;;</string>
     </property>
     <property name="text">
      <string>
welcome to based weather app
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>311</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 900 14pt &quot;Segoe UI Black&quot;;</string>
     </property>
     <property name="text">
      <string>
welcome to based weather app
</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>291</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(14, 30, 44);
border-radius:40px;
</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 900 11pt &quot;Segoe UI Black&quot;;</string>
     </property>
     <property name="text">
      <string>City :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_select">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>130</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: rgb(81, 43, 212);
color: rgb(255, 255, 255);
}


QPushButton:hover{
color: rgb(255, 255, 255);
background-color: rgb(27, 37, 212);
}</string>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="result_board">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>115</width>
       <height>81</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>115</width>
       <height>20</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Britannic Bold</family>
       <pointsize>14</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>96</width>
       <height>96</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>_____________________________________________________________</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cityName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>181</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>328</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
